lib/membrane/core/child/pads_specs.ex

```diff
@@ -246,15 +246,23 @@ defmodule Membrane.Core.Child.PadsSpecs do
     config_doc =
       config
       |> Enum.map(&generate_pad_property_doc/1)
-      |> Enum.map_join("\n", fn {k, v} ->
-        "<tr><td>#{k}</td> <td>#{v}</td></tr>"
+      |> Enum.map_join("\n", fn
+        {k, v} ->
+          if String.contains?(v, "\n") do
+            "#{k}:\n#{v}"
+          else
+            "#{k}: | #{v}"
+          end
+
+        nil ->
+          ""
```

Comment:
```suggestion
|> Enum.reject(&match?({:name, _name}, &1)
|> Enum.map_join("\n", fn -> 
  {:accepted_formats_str, formats} ->
    formats =       
        Enum.map_join(
          formats,
          &"""
          ```
          #{&1}
          ```
          """
        )
    "#{k}:\n#{v}"

  {property, value} ->
    property =  
      property |> to_string() |> String.replace("_", " ") |> String.capitalize()          
    "#{property}: | #{`#{inspect(value)}`}"
end)
```
Maybe do something like this and delete  `|> Enum.map(&generate_pad_property_doc/1)` from line 248? I guess, that only case, when `v` in line 251 contains `"\n"` is case with `accepted_format`. You can delete `generate_pad_property_doc` as well.

Reply:
If there are some `"\n"` in other cases than `:accepted_format_str`, ignore this suggestion